raco/myrial/scanner.py

```diff
@@ -17,7 +17,7 @@
 
 builtins = ['EMPTY', 'WORKER_ID', 'SCAN', 'COUNTALL', 'COUNT', 'STORE',
             'DIFF', 'CROSS', 'JOIN', 'UNIONALL', 'INTERSECT', 'DISTINCT',
-            'LIMIT', 'SINK']
+            'LIMIT', 'SINK', 'SAMPLEWRSCAN', 'SAMPLEWORSCAN']
```

Comment:
I considered making something like samplescan(relation, 500, WR) and samplescan(relation, 500, WoR). Would it be possible to detect the strings 'WR' or 'WoR' in the parser?